src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -56,3 +60,14 @@ def get(self, svc_id):
             'osd_metadata': mgr.get_metadata('osd', svc_id),
             'histogram': histogram,
         }
+
+
+@ApiController('osd/scrub')
+@AuthRequired()
+class OsdScrub(RESTController):
+    def create(self, osds, deep_scrub):
+        api_scrub = "osd deep-scrub" if deep_scrub else "osd scrub"
+
+        for osd in osds:
+            CephService.send_command("mon", api_scrub, who=osd)
```

Comment:
I think you need to convert the OSD ID to a string here to prevent the error message i posted before. This is working in the HostScrub where osd['id'] is a string.

Reply:
OK, can confirm that ``CephService.send_command("mon", api_scrub, who=str(osd))`` will fix the issue.

Reply:
Done.